.github/workflows/release.yml

```diff
@@ -197,11 +207,14 @@ jobs:
           if [ -n "$VERSION" ]; then
             sed -i '/^version /s/=.*$/= "'"$VERSION"'"/' hf_xet/Cargo.toml
           fi
+      - uses: ./.github/actions/set-build-profile
+        with:
+          tag: ${{ github.event.inputs.tag }}
       - name: Build wheels
         uses: PyO3/maturin-action@v1
         with:
           target: ${{ matrix.platform.target }}
-          args: --release --out dist
+          args: --profile ${{ env.BUILD_PROFILE }} --out dist
```

Comment:
Unfortunately, we do want to build with `--profile release` for windows. The way that the debug symbols are created are different for the different platforms.

You'll likely have to update the `set-build-profile` action to account for this.

Reply:
I'll just have windows not call the set-build-profile action.